src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap

```diff
@@ -103,7 +103,7 @@ exports[`LinksToItem renders "links and references" view 1`] = `
             className=""
           >
             <a
-              className="source published"
+              className="external source published"
               href="http://localhost:3000/page-cucumber"
```

Comment:
Why are links to localhost:3000 showing as external?

Reply:
@sneridagh waiting for your check result.

Reply:
Fixed.